README.md

```diff
@@ -1,3 +1,56 @@
-# Backend app
+# Music Library
+
+This is an app designed to perform CRUD requests on a locally-hosted PostgreSQL database. 
+
+## Installation
+
+Pull and run a docker image of PostgreSQL:
+
+`docker run --name postgres -p 5432:5432 -e POSTGRES_PASSWORD=password -d postgres`
+
+Install the project to your machine:
+
+git clone `https://github.com/JordsCodes/music-library`
+
```

Comment:
the user would also need to run `npm install` to add all of the project dependencies and also create .env and .env.test files. It's good to push a .env.example file to the GitHub repo so the user can see what sort of things to include